test/run-test.sh

```diff
@@ -14,7 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
+# This creates and deletes a namespace containing the given task name and version to execute the tests as closely to
+# CI as possible. As a consequence you will need to be cluster admin or require similar permissions.
```

Comment:
nit: "to be **a** cluster admin or **to have** similar permissions."

Reply:
updated